x-pack/legacy/plugins/canvas/canvas_plugin_src/uis/arguments/axis_config/__examples__/extended_template.stories.tsx

```diff
@@ -7,7 +7,7 @@
 import { action } from '@storybook/addon-actions';
 import { storiesOf } from '@storybook/react';
 import React from 'react';
-import { ExpressionAST } from '../../../../../types';
+import { ExpressionAstExpression as ExpressionAST } from '../../../../../types';
```

Comment:
This feels like a very strange name, and refactor... I assume VSCode simply used an `as` here to prevent churn... but `ExpressionAstExpression` feels redundant.  I would prefer these get changed to not use the `as`.

Reply:
Done. I've switched to `ExpressionAstExpression` in this file and other files.